### DISCUSSION TITLE
Welcome to Sorting Discussions!

### DISCUSSION THREAD

**Discussion by roarin-roran** [roarin-roran/Sorting#64] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by sebawild** [comment]
How shall we draw the line between issues and discussions? Smells like double accounting.

**Reply by roarin-roran** [reply]
sounds wise - my intention was to use discussions for questions about the code's direction, things that may or may not need to be changed. discussions are resolved with a decision, or with understanding. issues on the other hand are planned actions, whether that's a feature to add or a problem  to solve  -   they're always resolved by changes to the code base
